lib/kbsecret/cli/kbsecret-cp

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "kbsecret"
+require "tty-prompt"
+
+include KBSecret
+
+cmd = CLI.create do |c|
+  c.slop do |o|
+    o.banner = <<~HELP
+      Usage:
+        kbsecret cp [options] <target-session> <record> [record]
+    HELP
+
+    o.string "-s", "--session", "the session containing the record", default: :default
```

Comment:
I'm a little divided on whether the source session should be an option or an argument.On one hand it gives us a handy default session, but on the other it may not make sense to provide a default for this behavior (and be extra explicit about where we're copying to and from).

What do you think of an invocation like this? 

```
$ kbsecret cp src-dession dst-session record
```

As a more general note, we should definitely check to make sure that the source and destination sessions are different -- copying a record into the same session shouldn't have any effect, but a user shouldn't ever need to do it.

Reply:
You're right. This api looks cleaner.